### DISCUSSION TITLE
Phidgets or Yoctopuce?

### DISCUSSION THREAD

**Discussion by davidm870** [artisan-roaster-scope/artisan#1345] [Q&A] [answered]
Good $GREETINGTIME
We are preparing to add telemetry to some roasters and move from [REDACTED] to Artisan Scope. We're wondering if the community has any thoughts about whether the choice between the two thermocouple interfaces is a critical one. Are there any aspects of one or the other manufacturer's product that make it a better fit for Artisan? Thanks in advance for a response.

**Comment by MAKOMO** [comment, marked as answer]
Both, Phidgets and Yoctopuce modules are well supported by Artisan. You choice should be between RTDs and TCs. There are complete sets of Phidget-based [2xRTD](https://shop.artisan.plus/shop/en/hardware/phidgets-2xrtd-set) [(low noise)](https://artisan-roasterscope.blogspot.com/2019/03/on-idle-noise.html) and [2xTC](https://shop.artisan.plus/shop/en/hardware/phidget-2xtc-set) available in the artisan.plus shop. It would be of interest from what you move (REDACTED) or what machines you exactly have as Artisan [supports many machines out of the box](https://artisan-scope.org/machines/index) without any additional hardware and more machines are to be supported soon.

**Reply by davidm870** [reply]
I can't say much more than I already have at the moment, but we hope to significantly increase the use of precision roasting (via Artisan Scope) in parts of Latin America. Traditionally, high-quality coffee was reserved for export. This is changing and we're hoping to be part of the change.

I'm also continuing to try to adapt Artisan Scope to monitor the production of [biochar](https://biochar-international.org/). Changing the graph to accommodate ~850° C was trivial, but the program freaks out when I want to extend the "roast" time to 2.5 hours (not entirely unexpectedly 🤣).
